.github/workflows/main.yml

```diff
@@ -4,13 +4,13 @@ on:
   workflow_dispatch:
 
 jobs:
-  unit-tests:
-    name: unit tests
+  tests:
+    name: tests
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        php: [ '8.0', '8.1' ]
+        php: [ '8.1' ]
```

Comment:
@neomerx Note that you don't test for 8.0 compatibility now.
If that's not an issue, consider updating the minimum required php version in composer.json.

Reply:
I simply dropped it) I'm so glad you helped with it. Thank you very much.

Reply:
php 8.0 tests locally just fine but has an issue on Linux. The problems seems to be on Symfony side and maybe linux specific. I would just wait for fix.

Reply:
It's not the tests but the code coverage upload that's causing an issue. See the failed workflow runs on my fork (https://github.com/sneakyvv/cors-psr7/actions/runs/3172333224/jobs/5166885449). You'll see that the tests indeed succeed, and that the code coverage reports exists (`ls` call), but that the upload step fails.

Reply:
Exactly the same error in exactly the same Symfony file however during cs-fixer test
https://github.com/neomerx/cors-psr7/actions/runs/3190090634/jobs/5204724623
It doesn't seem to be related to the project and I would wait for some time and tried to return 8.0
Locally works just fine (though on Mac).